bindings/nodejs/lib/types/unlock_condition.ts

```diff
@@ -0,0 +1,175 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { Address, AliasAddress } from './address';
+
+/**
+ * All of the unlock condition types.
+ */
+enum UnlockConditionType {
+    Address = 0,
+    StorageDepositReturn = 1,
+    Timelock = 2,
+    Expiration = 3,
+    StateControllerAddress = 4,
+    GovernorAddress = 5,
+    ImmutableAliasAddress = 6,
+}
+
+abstract class UnlockCondition {
+    private type: UnlockConditionType;
+
+    constructor(type: UnlockConditionType) {
+        this.type = type;
+    }
+    /**
+     * The type of unlock condition.
+     */
+    getType(): UnlockConditionType {
+        return this.type;
+    }
+}
+
+class AddressUnlockCondition extends UnlockCondition /*implements IAddressUnlockCondition*/ {
+    private address: Address;
+    constructor(address: Address) {
+        super(UnlockConditionType.Address);
+        this.address = address;
+    }
+
+    /**
+     * The address.
+     */
+    getAddress(): Address {
+        return this.address;
+    }
+}
+/**
+ * Storage Desposit Return Unlock Condition.
+ */
+class StorageDepositReturnUnlockCondition extends UnlockCondition /*implements IStorageDepositReturnUnlockCondition*/ {
+    private amount: string;
+    private returnAddress: Address;
+
+    constructor(returnAddress: Address, amount: string) {
+        super(UnlockConditionType.StorageDepositReturn);
+        this.amount = amount;
+        this.returnAddress = returnAddress;
+    }
+    /**
+     * Amount of IOTA tokens the consuming transaction should deposit to the address defined in return address.
```

Comment:
Remove `IOTA tokens`, also the consuming transaction not only `should`, it must do it (unless expired)